frontends/web/src/routes/account/receive/receive-legacy.tsx

```diff
@@ -64,7 +62,6 @@ const getIndexOfMatchingScriptType = (
 export const Receive: FunctionComponent<Props> = ({
   accounts,
   code,
-  devices,
```

Comment:
the devices are still passed from receive/index.ts, right? typescript doesn't mind?

Reply:
I believe the short answer is it matches so TypeScript is happy.

https://stackoverflow.com/questions/49580725/is-it-possible-to-restrict-typescript-object-to-contain-only-properties-defined

https://github.com/Microsoft/TypeScript/issues/12936

Reply:
There is an eslint rule that would help, "react/no-unused-prop-types": "error" and catches a few cases, but unfortunately it has a few cases with the load decorator where it fails to recognise that those props are actually used and for some reason it does not catch this case :/

 I'll open a separate PR

Reply:
https://github.com/digitalbitbox/bitbox-wallet-app/pull/1733